utils/hct/hctdb.py

```diff
@@ -2138,6 +2141,47 @@ def UFI(name, **mappings):
             db_dxil_param(0, "i32", "", "number of levels of recursion remaining")])
         next_op_idx += 1
 
+        # Comparison Sampling
+        self.add_dxil_op("SampleCmpGrad", next_op_idx, "SampleCmpGrad", "samples a texture using a gradient and compares a single component against the specified comparison value", "hf", "ro", [
```

Comment:
Overloads for other `SampleCmp` ops are `"hfwi"`, should these be the same, or is this a mistake for the others?

Reply:
After some offline discussion. Tex and I have agreed that these were a mistake in the first place and I filed #5564